### PR TITLE
4040/launch with nextflow tower

### PR DESCRIPTION
**Pull request by NatalieEO**
https://user-images.githubusercontent.com/16004905/108127706-7fe7c880-7060-11eb-9b34-3034d9ed0e6b.mp4

dockstore/dockstore#4040

### CODE REVIEW

File: src/app/workflow/launch-third-party/launch-third-party.component.html [hunk 1]
```diff
@@ -107,6 +110,29 @@ <h3>Launch with</h3>
               ><img src="../assets/images/thirdparty/cavatica.png" /> Cavatica
             </a>
           </div>
+          <div fxFlex="100%" *ngIf="workflow?.descriptorType === 'NFL'">
+            <a
+              mat-raised-button
+              [matTooltip]="!(hasContent$ | async) ? 'The WDL has no content.' : 'Export this workflow to Nextflow Tower'"
```
Comment (coverbeck) [inline src/app/workflow/launch-third-party/launch-third-party.component.html:116]: WDL -> NextFlow

**Review by coverbeck — APPROVED** [review]
Is there an icon you could use?

**Comment by denis-yuen** [comment]
> Is there an icon you could use?

Open an issue on https://github.com/nextflow-io/trademark maybe

**Review by denis-yuen — APPROVED** [review]

**Comment by NatalieEO** [comment]
> > Is there an icon you could use?
> 
> Open an issue on https://github.com/nextflow-io/trademark maybe

Just sent them an email, but I thought we'd want the actual nextflow tower icon/logo and not the nextflow logo

File: src/app/workflow/launch-third-party/launch-third-party.component.html [hunk 2]
```diff
@@ -107,6 +110,29 @@ <h3>Launch with</h3>
               ><img src="../assets/images/thirdparty/cavatica.png" /> Cavatica
             </a>
           </div>
+          <div fxFlex="100%" *ngIf="workflow?.descriptorType === 'NFL'">
```
Comment (garyluu) [inline src/app/workflow/launch-third-party/launch-third-party.component.html:113]: There's an enum for NFL somewhere, could've used that here and up there.

**Review by garyluu — APPROVED** [review]
Approved either way

**Pull request merged**